src/components/Avatar/__tests__/Avatar.test.tsx

```diff
@@ -2,27 +2,28 @@ import React from 'react';
 import Avatar from '../Avatar';
 import { render, waitFor, configure } from '@testing-library/react';
 import { axe, toHaveNoViolations } from 'jest-axe';
+import colors from '../../../enums/colors';
 
 expect.extend(toHaveNoViolations);
 configure({ testIdAttribute: 'data-test-id' });
 const testId = 'foundry-avatar';
 
 describe('Avatar', () => {
   it('matches snapshot', async () => {
-    const { container } = render(<Avatar />);
+    const { container } = render(<Avatar color={colors.grayXlight} />);
```

Comment:
You can still default color to `colors.grayXlight`, just do it after the `useTheme` hook call